### DISCUSSION TITLE
PCOV extension

### DISCUSSION THREAD

**Discussion by lukas-frey** [beyondcode/herd-community#618] [General]
Hi, can we get the pcov extension by default?

When I want to test coverage, I either need to enable the xdebug extension or install pcov. But xdebug is enabled automatically in herd pro in the go in order to improve performance, so I guess pcov would be the best option to allow seamless integration for testing and coverage reports.

**Comment by girardotcl** [comment]
Hi,
I'm not able to install/config pcov on Herd.
How did you write your config in php.ini ?
